angular/src/app/modules/delivery-management/delivery/available-resource-tab/bill-account-plan/bill-account-dialog-note/bill-account-dialog-note.component.ts

```diff
@@ -41,6 +42,8 @@ export class BillAccountDialogNoteComponent implements OnInit {
           this.billNote = response.result.project.note;
           this.userId = response.result.userInfor.userId;
           this.projectId =response.result.project.projectId;
```

Comment:
ko dùng api this.planningBillInfoService.Get(this.data.userId,this.data.projectId) nữa
-> ở component cha truyền thêm projectName, fullName cho component con (popup)